src/cmd/create.js

```diff
@@ -0,0 +1,133 @@
+/* @flow */
+import path from 'path';
+import readline from 'readline';
+import tty from 'tty';
+
+import {fs} from 'mz';
+import mkdirp from 'mkdirp';
+import promisify from 'es6-promisify';
+
+import {createLogger} from '../util/logger';
+import {UsageError, isErrorWithCode} from '../errors';
+
+const log = createLogger(__filename);
+const defaultAsyncMkdirp = promisify(mkdirp);
+
+export type CreateParams = {|
+  dirPath: string,
+  stdin?: stream$Readable,
+|};
+
+export default async function create(
+  {
+    dirPath,
+    stdin = process.stdin,
+  }: CreateParams
+): Promise<void> {
+  const targetPath = path.join(process.cwd(), dirPath);
```

Comment:
I'm thinking that instead of using `process.cwd()` directly here, it would be better to turn it into a parameter (which defaults to process.cwd() if missing) and/or into an injected dependency (e.g. something like `function getBaseDir(baseDir) { return baseDir || process.cwd(); }`).

This way, the command can provide much more control to the caller if needed, which should be helpful both in the unit tests and while using web-ext as a library.